releases.md

```diff
@@ -1,14 +1,14 @@
 # Release Notes
 
 <!-- MarkdownTOC autolink="true" bracket="round" -->
-
+- [2019 December 18: Improved handling](#2019-december-18-improved-handling)
+- [2019 December 17: User interface improvements](#2019-december-17-user-interface-improvements)
 - [2019 October 25: Better discoverability](#2019-october-25-better-discoverability)
 - [2019 October 17: Bug fixes and regular maintenance for Go](#2019-october-17-bug-fixes-and-regular-maintenance-for-go)
 - [2019 October 9: More UI improvements](#2019-october-9-more-ui-improvements)
 - [2019 October 7: New Metrics Panel and New Go Center Look](#2019-october-7-new-metrics-panel-and-new-Go-Center-look)
 - [2019 September 19: SEO improvements](#2019-september-19-seo-improvements)
 - [2019 September 10: Checksum database support in GoCenter](#2019-september-10-checksum-database-support-in-gocenter)
-- [2019 September 3: Beginning to build a Go Modules Score](#2019-september-3-beginning-to-build-a-go-modules-score)
```

Comment:
Why was this entry removed?

Reply:
Right now it's missing the explanation - which may have been removed in a previous PR. Perhaps we keep it and link it to the modules scoring wiki here? https://github.com/jfrog/gocenter/wiki/Module-Scoring

Reply:
yep, makes sense

Reply:
Fixing now